internal/serve/publicfiles/js/receiver_registration.js

```diff
@@ -51,9 +51,9 @@ async function sendSms(phoneNumber, reCAPTCHAToken, onSuccess, onError) {
           recaptcha_token: reCAPTCHAToken,
```

Comment:
@quietbits or @ceciliaromao can you help review this part please ? I can't help much with JS. 

@ziyliu did we check with Tori about validation of these fields ? I think there should be some sort of front-end validation for this, especially the PIN.

Reply:
These updates look good to me. :+1:

Reply:
I forgot to add the backend validation too, but I did just now! I confirmed with Tori when I worked on the disbursement file validations about the requirements.